docs/source/nlp/quantization.rst

```diff
@@ -184,6 +184,32 @@ This script will produce a quantized ``.nemo`` checkpoint at the experiment mana
 It can also optionally produce an exported TensorRT-LLM engine directory or a ``.qnemo`` file that can be used for inference by setting the ``export`` parameters similar to the PTQ example.
 Note that you may tweak the QAT trainer steps and learning rate if needed to achieve better model quality.
 
+FP8-trained NeMo checkpoints
+---------------------------------
+
```

Comment:
Please mention somewhere that this needs to be FP8 Transformer Engine-trained checkpoint.

Reply:
I mean it should be immediately clear what is the input checkpoint, how it was obtained

Reply:
For example:

> which are already FP8-quantized

I would mention that they were pretrained/sft-ed with TE using FP8 rather than "FP8-quantized" (not really sure what it means)